### PR TITLE
Sense agents readme updates and simple CLI

### PR DESCRIPTION
**Pull request by Zaid-Waghoo**
Added a readme for how to use the Sensory Agent

### CODE REVIEW

File: mbodied/agents/sense/object_detection_agent.py [hunk 1]
```diff
@@ -32,12 +31,13 @@ def __init__(
             **kwargs,
         )
 
-    def act(self, image: Image, objects: list[str] | str, *args, api_name: str = "/detect", **kwargs) -> World:
+    def act(self, image: Image, objects: list[str] | str, model_type: str, *args, api_name: str = "/detect", **kwargs) -> World:
```
Comment (nqyy) [inline mbodied/agents/sense/object_detection_agent.py:34]: Give a default here

File: README.md [hunk 2]
```diff
@@ -254,7 +254,7 @@ To learn more about all of the possibilities with embodied agents, check out the
 - You can `pack` a list of `Sample`s or Dicts into a single `Sample` or `Dict` and `unpack` accordingly?
 - You can `unflatten` any python structure into a `Sample` class so long you provide it with a valid json schema?
 
-<details> <summary><h2 style="display: inline-block;">Deep Dive</h2></summary>
+<summary><h2 style="display: inline-block;">Deep Dive</h2></summary>
```
Comment (nqyy) [inline README.md:257]: No need for this anymore.

**Review by nqyy — APPROVED** [review]
Make sure @sebbyjp review CLI

**Review by nqyy — APPROVED** [review]

**Review by nqyy — APPROVED** [review]

**Review by sebbyjp — APPROVED** [review]

File: mbodied/agents/sense/object_detection_agent.py [hunk 3]
```diff
@@ -50,12 +59,18 @@ def act(self, image: Image, objects: list[str] | str, *args, api_name: str = "/d
...
+        )
         return World.model_validate(json_dict)
 
 
 # Example usage:
 if __name__ == "__main__":
     agent = ObjectDetectionAgent(model_src="https://api.mbodi.ai/sense/")
-    result = agent.act(image=Image("resources/bridge_example.jpeg", size=(224, 224)), objects=["spoon", "bowl"])
+    result = agent.act(
+        image=Image("resources/bridge_example.jpeg", size=(224, 224)),
```
Comment (sebbyjp) [inline mbodied/agents/sense/object_detection_agent.py:72]: files("mbodied") / resources

**Review by sebbyjp — DISMISSED** [review]

File: mbodied/agents/sense/object_detection_agent.py (hunk 3 above)
Comment (nqyy) [reply mbodied/agents/sense/object_detection_agent.py:72]: The resources is in root directory. Don't want to move them all yet. Lots of dependencies. They should be in separate PR.

**Pull request merged**